internal/services/formatters/python/safety/formatter_test.go

```diff
@@ -76,9 +80,14 @@ func TestFormatter_StartSafety(t *testing.T) {
 	t.Run("Should execute analysis without error", func(t *testing.T) {
 		analysis := getAnalysis()
 
-		config := &cliConfig.Config{}
-		config.WorkDir = &workdir.WorkDir{}
-		output := `{"issues": [{"dependency": "jinja2","vulnerable_below": "2.7.2","installed_version": "2.7.2","description": "The default configuration for bccache.FileSystemBytecodeCache in Jinja2 before 2.7.2 does not properly create temporary files, which allows local users to gain privileges via a crafted .cache file with a name starting with __jinja2_ in /tmp.","id": "123"}]}`
+		config := cliConfig.New()
+		config.ProjectPath = t.TempDir()
+		srcProject := testutil.PythonExample
+		dstProject := filepath.Join(config.ProjectPath, ".horusec", analysis.ID.String())
+		assert.NoError(t, copy.Copy(srcProject, dstProject, func(src string) bool {
+			print(src)
```

Comment:
Maybe we should use t.Logf()? The builtin print is never used in any place.

Reply:
done :)